src/main/webapp/app/entities/project/project-detail.tsx

```diff
@@ -126,6 +136,26 @@ const ProjectDetail: React.FC<IProjectDetailProps> = (props) => {
                     : null}{' '}
                 </dd>
               </dl>
+
+
+              <Chart
+                width={'500px'}
+                height={'300px'}
+                chartType="AreaChart"
+                loader={<Spinner color="primary" />}
+                data={diagramStatus}
+                options={{
+                  title: 'Project Activity',
```

Comment:
As everything is english right now, we can keep it like this but we should use i18n here as soon we support multiple languages.

Reply:
Fixed (:+1: